### PR TITLE
Follow coding conventions.

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by WinProtoTest** [comment]
Thanks for the pull request. We'll review it as soon as possible.

File: ProtoSDK/MS-KILE/KileDecrypter.cs [hunk 1]
```diff
@@ -241,18 +241,18 @@ public void ApExchange(byte[] apReq, byte[] apRep)
...
+                throw new ArgumentNullException(nameof(apReq));
             }
             if (apRep == null)
             {
-                throw new ArgumentNullException("apRep");
+                throw new ArgumentNullException(nameof(apRep));
             }
 
-            KileApRequest apReqPdu = new KileApRequest(kileDecoder.serverContext);
+            var apReqPdu = new KileApRequest(kileDecoder.serverContext);
```
Comment (JiajunW) [inline ProtoSDK/MS-KILE/KileDecrypter.cs]: We explicitly declare the type of variable, this is by design.

**Comment by JiajunW** [comment]
BTW, the `nameof` improvements could also be applied to other source code in MS-KILE SDK. Maybe you would like to check other files like KileRole.cs?

File: ProtoSDK/MS-KILE/KileDecrypter.cs (hunk 1 above)
Comment (Youssef1313) [reply ProtoSDK/MS-KILE/KileDecrypter.cs]: Look at the conventions [here](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)

> Use implicit typing for local variables when the type of the variable is obvious from the right side of the assignment, or when the precise type is not important.

**Comment by Youssef1313** [comment]
> 
> 
> BTW, the `nameof` improvements could also be applied to other source code in MS-KILE SDK. Maybe you would like to check other files like KileRole.cs?

@JiajunW , I've added the `nameof` improvement in the other files in `MS-KILE`.
Waiting for your reply about the `var` whether to add it to the other files or not.

**Comment by JiajunW** [comment]
@Youssef1313 Please go ahead. For obvious type, using `var` is reasonable.

**Comment by Youssef1313** [comment]
> 
> 
> @Youssef1313 Please go ahead. For obvious type, using `var` is reasonable.

Done

**Comment by JiajunW** [comment]
Thanks for your contribution!

BTW, `master` is the release branch. I have changed the base to `staging` branch.

**Pull request merged**